### PR TITLE
Simplified `Vec` import in stdl

### PR DESCRIPTION
**Pull request by Sofia-Insa**
Simplified `Vec` import in stdl
in `Data.Vec.Bounded` still import `Data.Vec` for `filter`, `takeWhile` and `dropWhile`

### CODE REVIEW

File: fix-whitespace.yaml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-included-dirs:
```
Comment (JacquesCarette) [inline fix-whitespace.yaml:1]: this seems like an odd change to happen in this PR?

File: src/Data/Vec/Bounded.agda [hunk 2]
```diff
@@ -10,7 +10,8 @@ module Data.Vec.Bounded where
 
 open import Level using (Level)
 open import Data.Nat.Base
-open import Data.Vec as Vec using (Vec)
+open import Data.Vec.Base using (Vec)
+import Data.Vec as Vec using (filter; takeWhile; dropWhile)
```
Comment (JacquesCarette) [inline src/Data/Vec/Bounded.agda:14]: As `Data.Vec` is still imported, probably revert this one.

File: src/Data/Vec/Membership/Propositional/Properties.agda [hunk 3]
```diff
@@ -10,8 +10,8 @@ module Data.Vec.Membership.Propositional.Properties where
 
 open import Data.Fin.Base using (Fin; zero; suc)
 open import Data.Product as Prod using (_,_; ∃; _×_; -,_)
-open import Data.Vec hiding (here; there)
 open import Data.Vec.Relation.Unary.Any using (here; there)
+open import Data.Vec hiding (here; there)
```
Comment (JacquesCarette) [inline src/Data/Vec/Membership/Propositional/Properties.agda:14]: This breaks the 'alphabetical order' where no characters is less than any character. And also, there is no effective change. Please revert.

(Even better would be to remove the `hiding` and put in a proper `using`)

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

**Review by gallais — APPROVED** [review]

File: src/Data/Nat/Binary/Subtraction.agda [hunk 4]
```diff
@@ -17,8 +17,8 @@ open import Data.Nat.Binary.Base
 open import Data.Nat.Binary.Properties
 import Data.Nat.Properties as ℕₚ
 open import Data.Product using (_×_; _,_; proj₁; proj₂; ∃)
-open import Data.Sum using (inj₁; inj₂)
 open import Data.Vec.Base using ([]; _∷_)
```
Comment (JacquesCarette) [inline src/Data/Nat/Binary/Subtraction.agda:20]: looks like the alphabetical ordering was inverted here?

File: src/Data/Vec/Bounded.agda [hunk 5]
```diff
@@ -10,9 +10,8 @@ module Data.Vec.Bounded where
 
 open import Level using (Level)
 open import Data.Nat.Base
-open import Data.Vec.Base using (Vec)
-import Data.Vec as Vec using (filter; takeWhile; dropWhile)
-open import Function
+open import Data.Vec as Vec using (Vec)
```
Comment (JacquesCarette) [inline src/Data/Vec/Bounded.agda]: This looks like a regression, i.e. an import of a super-module where a sub-module used to be used?

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

**Comment by MatthewDaggitt** [comment]
Thanks for the catches @JacquesCarette. Bad merge on my part

**Review by JacquesCarette — APPROVED** [review]

File: src/Tactic/RingSolver/Core/Polynomial/Semantics.agda [hunk 6]
```diff
@@ -14,7 +14,7 @@ module Tactic.RingSolver.Core.Polynomial.Semantics
   where
 
 open import Data.Nat          using (ℕ; suc; zero; _≤′_; ≤′-step; ≤′-refl)
-open import Data.Vec          using (Vec; []; _∷_; uncons)
+open import Data.Vec.Base          using (Vec; []; _∷_; uncons)
```
Comment (Taneb) [inline src/Tactic/RingSolver/Core/Polynomial/Semantics.agda:17]: Alignment here has gone squiffy.

**Review by Taneb — CHANGES_REQUESTED** [review]

**Review by Taneb — APPROVED** [review]

**Pull request merged**